src/Microsoft.DotNet.Darc/src/Darc/Options/SubscriptionsCommandLineOptions.cs

```diff
@@ -27,8 +27,8 @@ abstract class SubscriptionsCommandLineOptions : CommandLineOptions
         [Option("target-branch", HelpText = "Filter by target branch (matches substring unless --exact or --regex is passed).")]
         public string TargetBranch { get; set; }
 
-        [Option("frequencies", Separator = ',', Default = new string[] { "everyWeek", "twiceDaily", "everyDay", "everyBuild", "none", },
-            HelpText = @"Filter by subscription update frequency.")]
+        [Option("frequencies", Separator = ',',
+            HelpText = @"Filter by subscription update frequency. Valid values: ""everyWeek"", ""twiceDaily"", ""everyDay"", ""everyBuild"", ""none""")]
```

Comment:
What makes sure that the right values are being passed now?

Reply:
Nothing.  It's just a filter.  I'll change the help around to make it more clear, and change the filter code so that it matches on substring (e.g. you can type "week")